--- src/runner.rb	2017-11-05 00:04:33.699217269 +0000
+++ src/runner.rb	2017-11-05 11:31:29.096153933 +0000
@@ -119,7 +119,6 @@
         '--security-opt=no-new-privileges',  # no escalation
         "--ulimit data=#{gb4}:#{gb4}",       # max data segment size
         '--ulimit core=0:0',                 # max core file size
-        '--ulimit cpu=10:10',                # max cpu time (seconds)
         "--ulimit fsize=#{mb16}:#{mb16}",    # max file size
         '--ulimit locks=128:128',            # max number of file locks
         '--ulimit nofile=128:128',           # max number of files
