--- src/runner_scala.rb	2017-11-04 20:27:19.509937639 +0000
+++ src/runner.rb	2017-11-04 20:26:51.309776345 +0000
@@ -118,7 +118,7 @@
         '--security-opt=no-new-privileges',  # no escalation
         "--ulimit data=#{gb4}:#{gb4}",       # max data segment size
         '--ulimit core=0:0',                 # max core file size
-        '--ulimit cpu=10:10',                # max cpu time (seconds)
+        '--ulimit cpu=30:30',                # max cpu time (seconds)
         "--ulimit fsize=#{mb16}:#{mb16}",    # max file size
         '--ulimit locks=128:128',            # max number of file locks
         '--ulimit nofile=128:128',           # max number of files
